keyboards/paulpcb/info.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
If this firmware flashes as intended to your PCB as far as key assignments, then this file is wrong.

![image](https://user-images.githubusercontent.com/18669334/148462546-c2c49de5-1a91-4fcf-baa7-03dbd13cb034.png)

The JSON objects need to be in the same order that `keymap.c` uses.